x/protorev/keeper/epoch_hook_test.go

```diff
@@ -0,0 +1,74 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	"github.com/osmosis-labs/osmosis/v13/x/protorev/types"
+)
+
+func (suite *KeeperTestSuite) TestEpochHook() {
+	// All of the pools initialized in the setup function are available in keeper_test.go
+	// akash <-> types.OsmosisDenomination
+	// juno <-> types.OsmosisDenomination
+	// ethereum <-> types.OsmosisDenomination
+	// bitcoin <-> types.OsmosisDenomination
+	// canto <-> types.OsmosisDenomination
+	// and so on...
+	expectedToSee := make(map[string]Pool)
+	for _, pool := range suite.pools {
+
+		// Module currently limited to two asset pools
+		// Instantiate asset and amounts for the pool
+		if len(pool.PoolAssets) == 2 {
+			pool.Asset1 = pool.PoolAssets[0].Token.Denom
+			pool.Amount1 = pool.PoolAssets[0].Token.Amount
+			pool.Asset2 = pool.PoolAssets[1].Token.Denom
+			pool.Amount2 = pool.PoolAssets[1].Token.Amount
+		}
+
+		if pool.Asset1 == types.OsmosisDenomination || pool.Asset2 == types.OsmosisDenomination || pool.Asset1 == types.AtomDenomination || pool.Asset2 == types.AtomDenomination {
+			// create a key that is a combination of asset1 and asset2 in alphabetical order
```

Comment:
Note: We started sorting denoms lexicographically in twap so might wanna see if it's relevant here

Reply:
pools are a custom struct that we made in `keeper_test.go` so we can't assume that they are sorted. However, we will keep this in mind for subsequent PRs (if we sort by pool asset anywhere, we will just assume its already done for us).